### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by zenangst**
This PR adds the following cells:
### List
- [x] ButtonListView
- [x] CoreListView
- [x] ImageListView
- [x] InfoListView
- [x] InputListView
- [x] ListView
- [x] ToggleListView
### Grid
- [x] CoreGridView

Keep in mind that these views are still under heavy refactoring at the moment. We will all have to help to get them ready for production when the time comes.

This PR also sets up a basic demo to show different variations of the views that is included with `Masquerade`.

### CODE REVIEW

**Comment by zenangst** [comment]
Oh, first PR, such joy!

When reviewing, think about that not all views are optimized.

File: Sources/Shared/Item+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+public extension Item {
+
+  var styles: String {
+    set { meta["styles"] = newValue }
+    get { return meta.property("styles") ?? "" }
+  }
+
+  var imageHeight: CGFloat {
+    set { meta["image-image"] = newValue }
```
Comment (vadymmarkov) [inline Sources/Shared/Item+Extensions.swift]: @zenangst I guess it should be "image-height"?

File: Sources/iOS/InputListView.swift [hunk 2]
```diff
@@ -0,0 +1,199 @@
+import Brick
+import Spots
+import Tailor
+import UIKit
+
+public enum InputFieldAlignment: String {
+  case center = "center", left = "left"
```
Comment (vadymmarkov) [inline Sources/iOS/InputListView.swift]: Don't we get strings that reflect name for free?

File: Sources/Shared/Item+Extensions.swift (hunk 1 above)
Comment (zenangst) [reply Sources/Shared/Item+Extensions.swift]: Holy crap Batman, you are totally right!

File: Sources/iOS/InputListView.swift (hunk 2 above)
Comment (zenangst) [reply Sources/iOS/InputListView.swift]: Do we? I think the alignment resolves to `Int` values.

Comment (zenangst) [reply Sources/iOS/InputListView.swift]: Oh... you mean like that!....

Comment (zenangst) [reply Sources/iOS/InputListView.swift]: Tihi, I think this is post `swift-3`.

**Comment by vadymmarkov** [comment]
@zenangst Great work!

**Comment by zenangst** [comment]
Ich :)

**Pull request merged**